core-spi-test/src/main/java/org/ehcache/internal/tier/CachingTierGetOrComputeIfAbsent.java

```diff
@@ -96,7 +96,7 @@ public void returnTheValueHolderCurrentlyInTheCachingTier() throws LegalSPITeste
     when(computedValueHolder.value()).thenReturn(value);
     when(computedValueHolder.expirationTime(any(TimeUnit.class))).thenReturn(Store.ValueHolder.NO_EXPIRE);
 
-    tier = factory.newCachingTier(1L);
+    tier = factory.newCachingTier();
```

Comment:
Yes, with some of the fixes around the bytesized on heap, this caused issues. And the size is not really a factor unless you test eviction.